includes/functions.php

```diff
@@ -980,8 +984,11 @@ function tsml_get_meeting($meeting_id=false) {
 			'saddr' => 'Current Location',
 			'q' => $meeting->location,
 		));
+		$meeting->address_approximate = $location->approximate;
```

Comment:
for the purpose of the spec, the value that goes into JSON should be `true` or `false` -- could we make this 

```
$meeting->address_approximate = (!empty($location->approximate) && $location->approximate === 'yes');
```

Reply:
Some other code will have to be updated as well, to make it use the approximate field as a boolean, rather than a string of 'yes' or 'no', but doable